src/dodal/beamlines/i22.py

```diff
@@ -309,3 +310,16 @@ def linkam(
         wait_for_connection,
         fake_with_ophyd_sim,
     )
+
+
+@skip_device
+def ppump(
```

Comment:
Is this deliberately double p? If so what does the first p stand for?

Reply:
Interjecting here, the double p matches what's on i22's synoptic (it feels like matching the synoptic name <-> bluesky name makes sense?).

The p is for peristaltic pump I think, as opposed to spump which is a syringe pump I believe.

Reply:
Can we put this in the docstring? Otherwise people will (rightly) think it looks like a typo

Reply:
Or could call the device `peristaltic_pump`

Reply:
Added docstring description in 6d5ae3fb866bdad8de27966fc11605b6c41856cd, I'll find out what the scientists would prefer as the device name.